aten/src/ATen/native/GatedLinearUnit.cpp

```diff
@@ -93,6 +94,49 @@ Tensor glu_jvp(
   return dglu;
 }
 
+Tensor glu_vjp(
+    const Tensor& grad_x,
+    const Tensor& grad_glu,
+    const Tensor& x,
+    const Tensor& dgrad_glu,
+    const Tensor& dx,
+    int64_t dim
+) {
+  dim = maybe_wrap_dim(dim, x.dim());
+  const auto glu_size = grad_glu.size(dim);
+  const auto a = x.narrow(dim, 0, glu_size);
+  const auto b = x.narrow(dim, glu_size, glu_size);
+  const auto da = dx.narrow(dim, 0, glu_size);
+  const auto db = dx.narrow(dim, glu_size, glu_size);
+  // grad_x_a = grad_glu * sigmoid(b)
+  const auto grad_x_a = grad_x.narrow(dim, 0, glu_size);
+  // grad_x_b = grad_x_a * a * (1 - sigmoid(b))
+  const auto grad_x_b = grad_x.narrow(dim, glu_size, glu_size);
+
+  const auto sig_b = at::sigmoid(b);
+  // TODO: use glu from forward.
```

Comment:
Good idea, not in this PR though right?

Reply:
No, too much work, since `glu_backward` is implemented to bell inf-many times differentiable, so adding a new argument will require touching all diff-related functions.

Reply:
Yup changing the signature does seem like a good amount of work here